### ISSUE TITLE
Link “Let’s work on your site” to Launchpad.

### ISSUE THREAD

**Issue by TimBroddin** [Automattic/wp-calypso#85197]
I don’t know if we can check from there if the site still has a Launchpad open and link to it?

![Image](https://github.com/Automattic/wp-calypso/assets/528287/c528a231-375e-463c-b68b-38a0596464ea)

This task will need some exploration.

**Comment by TimBroddin** [comment]
We're currently sending them to /home, which should have the biggest chance of having a Launchpad. What should we change @obenland ?

**Comment by obenland** [comment]
> Connect “A site refresh” box to site age.
Maybe we can only show this for sites that we don’t consider a new site anymore? [6] months old? For new sites, can we see if there’s a widget like that that covers setting up a site or getting the most out of a new plan on wp.com/learn?

In the walkthrough they had just created a site and were shown this "Refresh Site" CTA (pdtkmj-1Wb-p2#comment-3671).
I had a typo in my comment, but I wondered if we could make this CTA dependent on site age, and only show it for sites that we don't consider new anymore. I don't know if we have a definition for that yet, and in case we don't, suggested 6 months as a cutoff.
For new sites, I wonder if there is an existing CTA that points to a Learn resource that we could use instead.

**Comment by obenland** [comment]
> Finally, when I click on ‘lets work on the site’, it takes me to the Calypso homepage, rather the launchpad?
> —pdtkmj-1Wb-p2#comment-3671

It looks like the reporter's expectation was to maybe be redirected to the full-screen launchpad, rather than Calypso home. Is there a way we can check completion of that and link to it instead?

**Comment by TimBroddin** [comment]
This needs some further investigation.

**Comment by TimBroddin** [comment]
Fixed in: https://github.com/Automattic/wp-calypso/pull/86234

Waiting on Dale's review, who was AFK last week.
